requirements.txt

```diff
@@ -1,14 +1,14 @@
 # Switch into a virtual environment
 # pip install -r requirements.txt
--e .
+
 cython>=0.19
 dash>=1.6.1
 dash-bio-utils==0.0.4
 dash-daq==0.2.2
 gunicorn==19.9.0
 jsonschema==2.6.0
 matplotlib==3.0.2
-numpy==1.15.4
```

Comment:
I feel like I've asked this question before, but does this root `requirements.txt` file get used anywhere?

Reply:
This was originally used for the demo apps but I don't think it needs to be here any more after https://github.com/plotly/dash-bio/pull/459 since the apps each have their own `requirements.txt`. We should probably replace it with a `dev-requirements.txt` file instead.

Reply:
@shammamah Good to know, thanks! No need to fix now but please make an issue for followup.